soh/soh/Enhancements/randomizer/3drando/location_access/locacc_bottom_of_the_well.cpp

```diff
@@ -160,7 +181,11 @@ void RegionTable_Init_BottomOfTheWell() {
                   //You have to throw the pot from further back to hit the switch from the front instead of the top, trying to hit the "fingers" directly
                   //This unintuitiveness means it should be a trick. ZL is needed to get a clear path to carry the pot
                   EventAccess(&logic->LoweredWaterInsideBotw, {[]{return logic->CanJumpslash() || logic->CanUseProjectile();}}),
-  }, {}, {
+  }, {
+                  //Locations
+                  //Implies CanBreakPots()
```

Comment:
I'm honestly not sure. I thought it had to do with BlastOrSmash also considering having strength to pick up bombs, but that one just checks directly for having bombs or bombchus. @Pepper0ni Assuming you wrote this line, you want to chime in here?

Reply:
Basically, you need to break pots in addition to the stated conditions, but if you fulfill those conditions, you can break pots. Because of this I don't include the extra check for performance reasons, but leave the comment in case something weird happens in shuffles that I don't expect and it matters anyway.